### ISSUE TITLE
pytest historic database does not hadles exception on UI if user creates a project with name of database already available in MySQL

### ISSUE THREAD

**Issue by parwalrahul** [adiralashiva8/pytest-historic#8]
**Steps to reproduce:**
1. Navigate to pytesthistoric (localhost:5000)
2. Click in getting started
3. Click on new project
4. Enter Name as test and description as test.
5. Click Create

**Console Logs :**
![image](https://user-images.githubusercontent.com/12759282/76334209-b9746780-6318-11ea-99ca-a1880bffbec1.png)

(1007, "Can't create database 'test'; database exists")

**Comment by adiralashiva8** [comment]
Good catch :), I have to look into it. Until fix requesting users to not add duplicates in `Create Project` page
![image](https://user-images.githubusercontent.com/12430515/76335137-1290cb00-631a-11ea-972e-673e145c9adb.png)

**Comment by parwalrahul** [comment]
That's cool Shiva ! 
I found this issue thinking of using this for demo (poc ) mindset where user tries to create projects by the name of "test".
